### PR TITLE
List `json` command in README

### PR DESCRIPTION
**Pull request by nichtich**
Marked as unstable because of
<https://github.com/deutsche-nationalbibliothek/pica-rs/issues/249>.

### CODE REVIEW

**Comment by niko2342** [comment]
I've removed the `json` command in this table, because I'm not sure how to proceed with this command. On the one hand, this command causes irritation, because of the format. The picadata eco-system uses the so called [PICA JSON](http://format.gbv.de/pica/json) format, which is compact, but not readable. (I do not like this format). Also the IT department of the DNB is not happy, because they use different identifier ("name" instead of "tag", etc).

On the other hand, it serializes the internal data structure very efficient with a low memory footprint. This is very useful when serializing more than 100.000 records. Furthermore, it is very useful to have a descriptive format when deserialize json data in python (use strings in a dictionary).

In my opinion there shouldn't be a discussion about the "right" way to serialize PICA+ data in json. There are different requirements which lead to different json serializations. If we can all agree on this, I'll add the command in the README (and documentation) back. Otherwise this command gets a hidden feature, which must be enabled explicitly.

Notwithstanding the above, the `json` command is not unstable.

**Comment by nichtich** [comment]
If it has a use case, it should be documented. I only ask for a name to list and describe it at http://format.gbv.de/pica. and not to change the format once it has been established.

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ $ cargo install --git https://github.com/deutsche-nationalbibliothek/pica-rs --t
 | [select](#select)       | beta      | write subfields to a CSV file                                     |
 | [slice](#slice)         | beta      | return records withing a range (half-open interval)               |
 | [split](#split)         | beta      | split a list of records into chunks                               |
+| [json](#json)           | unstable  | serialize records in JSON                                         |
```
Comment (niko2342) [inline README.md:48]: Stability of the command is beta

File: README.md [hunk 2]
```diff
@@ -248,6 +222,33 @@ outdir
 ├── ...
 ```
 
+### JSON
+
+To serialize a record to JSON, just run the following command:
```
Comment (niko2342) [inline README.md:227]: I think the description should be clear about the resulting format. I propose sth. like this: This command serialize the internal representation of a record to JSON. The resulting JSON can be used to do subsequent data analysis (ex. in Python), but should not be used as an interchange format (use PicaJSON instead).

**Review by niko2342 — CHANGES_REQUESTED** [review]

**Comment by nichtich** [comment]
I've modified the description of JSON command and included a not on how to get PICA JSON, if needed.

**Review by niko2342 — APPROVED** [review]

**Comment by niko2342** [comment]
Thanks for your PR! Excellent idea to add the `jq`-code for converting between the different formats!

**Pull request merged**